qa/systemd-test/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply plugin: 'opensearch.standalone-rest-test'
+
+tasks.register("integTest", Test) {
```

Comment:
AFAIK you probably don't need to run this test as part of Gradle check, right?

Reply:
Nope, this only needs to be executed as part of the build integTest.